src/material/chips/chips.md

```diff
@@ -1 +1,194 @@
-<!-- TODO: this guide and the related live examples need to be rewritten to reflect the new chips API. -->
+Chips allow users to view and interact with a list of information.
```

Comment:
The new chip grid also allows the user to edit chips. Should we have a section and a live demo showing it? You can see some examples in the dev app demo by searching for "editable".

Reply:
That's a good point. This sentence was probably not detailed enough. I changed it to "Chips allow users to view information, make selections, filter content, and enter data." – it goes in the order that chips.md presents the different interaction patterns.

Reply:
I can edit the editing functionality to the chips-input example.

Reply:
I added editing functionality to chips-input demo